Main.java

```diff
@@ -10,6 +13,12 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    System.out.println("Please enter two numbers you would like to divide on seperate lines");
+    int firstNumber =input.nextInt();
+    int secondNumber =input.nextInt();
+    int finalNumber =firstNumber/secondNumber;
+    int remainderNumber= firstNumber % secondNumber;
+   System.out.println("The number is " + finalNumber +  " with a remainder of " + remainderNumber + ".");

```

Comment:
Please make sure you are commenting your code! The output is also off and doesn't match the sample, which is why it fails the tests. Otherwise, this is a good looking program!